screenshots/build.js

```diff
@@ -0,0 +1,17 @@
+var page = require('webpage').create();
+page.viewportSize = {width: 1000, height: 1000};
+page.open('./test/perceptual/alert.html', function() {
+	var clipRect = page.evaluate(function () {
+		var screenshot = document.querySelector(".screenshot");
+		screenshot.style.display = 'inline-block';
+		return screenshot.getBoundingClientRect();
+	});
+	page.clipRect = {
+		top:    clipRect.top,
+		left:   clipRect.left,
+		width:  clipRect.width,
+		height: clipRect.height
+	};
+	page.render('screenshot.png');
```

Comment:
With buttons have we been manually moving the screenshot into the screenshots folder? (We weren't sure if there was some sort of build integration that we weren't seeing that would change the working directory or something.)